### DISCUSSION TITLE
Why aren't you unblocking me? I have sent multiple emails with no response to unblocking my account, it's been a month! I'm getting really frustrated. (Email: helloguylolol2284@gmail.com)

### DISCUSSION THREAD

**Discussion by jhjhfjdhfjdhfjd** [vercel/community#4170] [Account Issue] [answered]
### Question

Soo.... please go to emails and just unblock me already.

**Comment by SkyElephone** [comment]
i have the same question!My account is blocked!

**Comment by hungchihsueh** [comment]
i have the same question!My account is blocked!

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. Tickets are processed by the order in which they are received, so it can take a bit of time for the team to get through all cases when there is a large amount of new tickets.

For more info, please check the following guides: 
- [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
- [How long does it take to get a response from Vercel Support?](https://vercel.com/guides/vercel-support-queue-time#how-vercel-support-works)

**Comment by SkyElephone** [comment]
Please help me,my account has blocked!I don't know!

Amy ***@***.***> 于 2023年9月25日周一 下午11:03写道：
